runtime/src/test_utils.rs

```diff
@@ -632,39 +604,17 @@ impl MockRuntime {
         self.epoch = epoch;
     }
 
-    pub fn expect_get_randomness_from_tickets(
-        &mut self,
-        tag: DomainSeparationTag,
-        epoch: ChainEpoch,
-        entropy: Vec<u8>,
-        out: Randomness,
-    ) {
-        let a = ExpectRandomness {
-            tag,
-            epoch,
-            entropy,
-            out,
-        };
+    pub fn expect_get_randomness_from_tickets(&mut self) {
+        let a = ExpectRandomness {};
         self.expectations
             .borrow_mut()
             .expect_get_randomness_tickets
             .push_back(a);
     }
 
     #[allow(dead_code)]
-    pub fn expect_get_randomness_from_beacon(
-        &mut self,
-        tag: DomainSeparationTag,
-        epoch: ChainEpoch,
-        entropy: Vec<u8>,
-        out: Randomness,
-    ) {
-        let a = ExpectRandomness {
-            tag,
-            epoch,
-            entropy,
-            out,
-        };
+    pub fn expect_get_randomness_from_beacon(&mut self) {
```

Comment:
Why do we need to keep these? It is imposed by the trait definition? If this is the case, which one? I feel we can remove this from the runtime.